slack_notifier.rb

```diff
@@ -0,0 +1,42 @@
+require "json"
+require "curb"
+
+class SlackNotifier
+  def initialize(slack_webhook)
+    @slack_webhook = slack_webhook
+  end
+
+  def new_issue_not_published
+    send_message("*Fíha, dnes na ÚVO nevyšlo nové vydanie vestníka?*")
+  end
+
+  def matching_announcements(page, announcements)
+    send_header_message(page, announcements)
+
+    announcements.each do |a|
+      send_message("<#{a[:link][:href]}|#{a[:link][:text]}>: *#{a[:customer]}* #{a[:description]}")
```

Comment:
Ad. Timecop - ono to vlastne s tym datumom neni vlastne taky blby napad. Lebo uvobota sa vlastne pytas - nasiel si pre 27.1.2016 nejake zakazky? To by potom pomohlo keby sme to chceli dotahat aj do historie.

Ad. `no_matching_announcements_found` vobec nie zly napad, presunie to logiku aj inde, test bude tiez krajsi :+1: